docs/contributing/gsoc.md

```diff
@@ -424,10 +424,12 @@ CSS](https://github.com/zulip/zulip/).
   settings/administration/streams overlays in a development
   environment. You can get experience working on the subsystem by
   working on some of [our open settings/admin
-  issues](https://github.com/zulip/zulip/labels/area%3A%20admin). 175
+  issues][filtered_issues]. 175
```

Comment:
Changed this to `all-settings-issues` -- we generally use readable names even for these references.